### ISSUE TITLE
Add step requirements related to categories.

### ISSUE THREAD

**Issue by tw2113** [opencredit/badgeos#403]
https://wordpress.org/support/topic/badge-for-all-comments-on-a-specific-category?replies=1

**Comment by tw2113** [comment]
Examples:
-Comment on post from X category.
-Comment on Y amount of posts from X category.

**Comment by daveharrisonnet** [comment]
Is this in place now as a feature? Is their any documentation?

**Comment by tw2113** [comment]
No, this is an issue holding ideas/details if we decide to implement.

**Comment by daveharrisonnet** [comment]
Ok Michael

thanks for the reply.
I really hope you do implement this functionality
Can you let me know if you do or if yo have a roadmap ETA

Dave

On 02/12/2015 15:41, Michael Beckwith wrote:
>
> No, this is an issue holding ideas/details if we decide to implement.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/opencredit/badgeos/issues/403#issuecomment-161337805>.
>

-- 

*Dave Harrison*

Owner & Principal Designer

Web: http://daveharrison.net <http://daveharrison.net/>

---- http://zerply.com/daveharrisonnet/public

---- http://linkedin.com/in/daveharrisonnet


Email: dave@daveharrison.net <mailto:dave@daveharrison.net>

Phone: 079 1445 7924


        *"I design and build effective websites"*

This e-mail is from Dave Harrison. It contains information which is 
confidential and which may be privileged. It is for the exclusive use of 
the intended recipient. Any dissemination, distribution, copying or use 
of this communication or any attachment to it without prior permission 
of the addressee is strictly prohibited. If you have received it in 
error, please inform me by e-mail or phone and then delete it from your 
system.

While I have taken reasonable precautions to minimise the risk of any 
attachment to this e-mail containing computer viruses, I cannot accept 
liability for any damage which you sustain as a result of any such 
viruses.You should carry out your own virus checks before you open this 
document.

**Comment by tw2113** [comment]
Nothing official at the present time.
